eng/common/pipelines/templates/archetype-typespec-emitter.yml

```diff
@@ -67,9 +67,14 @@ parameters:
   type: object
   default: []
 
-- name: EmitterPackageJsonPath
+# The path to the emitter package json file.
```

Comment:
These changes will be reverted once https://github.com/Azure/azure-sdk-tools/pull/10293 merges